### ISSUE TITLE
UI - Subscribe button appearance

### ISSUE THREAD

**Issue by chrismorel** [GShotwell/shambhala_app_issues#11]
The "Subscribe" button looks large, compared to rest of screen & other buttons.  In checking the height of the button, though, it is the same as the other 4 main buttons.  Maybe, it seems large because of the position of it or the dark blue background?

Suggestion - maybe, try other color, background, and/or size?  Check if others have a similar comment?

Tested on: iPad

**Comment by gshotwell** [comment]
Hmm, I think this is partially a question of whether we want to really be support tablets. It seems like we'll need different codebases for iPads and iPhones just because it's difficult to have things be legible on a small screen but not look too large on the big screen

**Comment by chrismorel** [comment]
Good point.  Note that in using TestFlight on the iPad, it does have 2 modes for the UI - smaller & larger.  The smaller is closer to a smart phone size and the larger is tablet size.  Good to idea to test for & emphasize phone screens.

Also, this is more of a subjective comment / suggestion :-)
